deploy.sh

```diff
@@ -29,32 +32,60 @@ function buildandpush {
   docker push schulcloud/schulcloud-client-n21:$GIT_SHA
   docker push schulcloud/schulcloud-client-open:$DOCKERTAG
   docker push schulcloud/schulcloud-client-open:$GIT_SHA
+  docker push schulcloud/schulcloud-client-brb:$DOCKERTAG
+  docker push schulcloud/schulcloud-client-brb:$GIT_SHA
 }
 
 function deploytotest {
   # build container default theme
-  docker build -t schulcloud/schulcloud-client:$DOCKERTAG -t schulcloud/schulcloud-client:$GIT_SHA .
+ # docker build -t schulcloud/schulcloud-client:$DOCKERTAG -t schulcloud/schulcloud-client:$GIT_SHA .
 
   # take those images and push them up to docker hub
-  docker push schulcloud/schulcloud-client:$DOCKERTAG
-  docker push schulcloud/schulcloud-client:$GIT_SHA
+ # docker push schulcloud/schulcloud-client:$DOCKERTAG
+ # docker push schulcloud/schulcloud-client:$GIT_SHA
 
   # screw together config file for docker swarm 
-  eval "echo \"$( cat compose-client-test.dummy )\"" > docker-compose-client.yml
+ # eval "echo \"$( cat compose-client-test.dummy )\"" > docker-compose-client.yml
 
   # copy config-file to server and execute mit travis_rsa
   chmod 600 travis_rsa
-  scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i travis_rsa docker-compose-client.yml linux@test.schul-cloud.org:~
-  ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i travis_rsa linux@test.schul-cloud.org /usr/bin/docker stack deploy -c /home/linux/docker-compose-client.yml test-schul-cloud
-  ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i travis_rsa linux@test.schul-cloud.org /usr/bin/docker service update --force test-schul-cloud_client
+#  scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i travis_rsa docker-compose-client.yml linux@test.schul-cloud.org:~
+#  ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i travis_rsa linux@test.schul-cloud.org /usr/bin/docker stack deploy -c /home/linux/docker-compose-client.yml test-schul-cloud
+  ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i travis_rsa linux@test.schul-cloud.org /usr/bin/docker service update --force --image schulcloud/schulcloud-client:develop test-schul-cloud_client
+}
+
+function deploytoprods {
+  chmod 600 travis_rsa
+  # open
+  ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i travis_rsa linux@open.schul-cloud.org /usr/bin/docker service update --force --image schulcloud/schulcloud-client-open:latest open_client
```

Comment:
Sollte das nicht per web-hook o.ä. gemacht werden, als von hier aus? Dann könnten die Zugangsdaten usw. irgendwo anders als in Travis liegen und ein böser Hackerman könte nicht das `build.sh`-file manipulieren. Oder denke ich hier völlig falsch? @FalkoBenthin @DavidJaeger-hpi

Reply:
Wie besprochen, wird evaluiert